run_uh.py

```diff
@@ -50,19 +50,15 @@
 # this will break any run_uh imports from other locations (e.g. _get_version())
 
 def exit_with_error(title, message):
-	if title + '.' != message:
-		print(title)
-	print(message)
-
+	print(title + '\n' + message);
 	try:
 		import Tkinter
 		import tkMessageBox
 		window = Tkinter.Tk()
 		window.wm_withdraw()
 		tkMessageBox.showerror(title, message)
 	except ImportError:
-		# tkinter may be missing
-		pass
+		print("Tkinter module missing. Install Tkinter for Python2.7 to view pop-up error dialogs");
```

Comment:
Yes, I agree that the need for an extra debugging message in `exit_with_error()` is somewhat debatable. As alternatives I see:
- placing a try-except at the beginning of the run_uh.py code, possibly in the section with all of the other imports
- add a note in the documentation ("If you want to see additional error pop-ups, make sure your Python2.7 was installed with Tkinter") -> **recommended as it's the simplest solution**

The thing is that Tkinter comes with Python on many, but not all platforms. For instance, on FreeBSD it's an optional module with all of the Python packages, but the base Python packages do not contain it. On the other hand, missing Tkinter doesn't break anything as the error message printed by `exit_with_error()` is sent to stdout anyhow. 

I will therefore remove it and restore the `pass` instruction.